rpath-link should be avoided on Solaris:
https://bugzilla.mozilla.org/show_bug.cgi?id=1365620

Adds -R,$ORIGIN to solve some dependency issue during gmake publish (not needed to run FF)
Should be more inverstigated.

--- firefox-52.1.0esr/old-configure.in.orig	2017-05-12 06:26:13.859030311 +0000
+++ firefox-52.1.0esr/old-configure.in	2017-05-12 06:27:43.071305404 +0000
@@ -1275,6 +1275,10 @@
     fi
     ;;
 
+*-solaris*)
+    MOZ_FIX_LINK_PATHS="-Wl,-R,'\$\$ORIGIN'"
+    ;;
+
 esac
 
 AC_SUBST_LIST(MMX_FLAGS)
--- firefox-52.1.0esr/js/src/old-configure.in.orig	2017-05-12 06:41:32.909798681 +0000
+++ firefox-52.1.0esr/js/src/old-configure.in	2017-05-12 06:42:53.309730714 +0000
@@ -931,6 +931,10 @@
     fi
     ;;
 
+*-solaris*)
+    MOZ_FIX_LINK_PATHS="-Wl,-R,'\$\$ORIGIN'"
+    ;;
+
 esac
 
 dnl Only one oddball right now (QNX), but this gives us flexibility
